### PR TITLE
[CTOR-438] [Doc] [CommunityPR] Infor ion grid restapi 

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Adds a new plugin to monitor Infor ION Grid components through its Rest API.
Development from C.Gagnaire https://github.com/centreon/centreon-plugins/pull/4944

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-infor-ion-grid-restapi.md [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+## Comment puis-je tester le plugin et que signifient les options des commandes ?
+
+Une fois le plugin installé, vous pouvez tester celui-ci directement en ligne
+de commande depuis votre collecteur Centreon en vous connectant avec
+l'utilisateur **centreon-engine** (`su - centreon-engine`). Vous pouvez tester
+que le connecteur arrive bien à superviser une ressource en utilisant une commande
+telle que celle-ci (remplacez les valeurs d'exemple par les vôtres) :
+
+```bash
+/usr/lib/centreon/plugins/centreon_infor_ion_grid_restapi.pl \
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-infor-ion-grid-restapi.md:239]: @cg-tw : In original community PR there are real examples for both modes application and node, do you prefer them or we can let this one ?

`centreon_plugins.pl --plugin=apps::infor::ion::grid::api::plugin --mode=application --hostname='toto.domain' --port='23456' --proto='https' --api-username='blabla' --api-password='(/ç*/&"*' --filter-name='^M3BE_15.1$' --warning-status='%{online} =~ /true/ && %{state} =~ /^NOT OK$/i' --critical-status='%{online} =~ /false/ || %{state} =~ /^NOT RUNNING$/i'
OK: Application 'M3BE_15.1' [description: System Foundation for M3 BE] state is 'OK' [online: true] [started: true]`

`centreon_plugins.pl --plugin=apps::infor::ion::grid::api::plugin --mode=node --hostname='toto.domain' --port='23456' --proto='https' --api-username='blabla' --api-password='(/ç*/&"*' --filter-name='^Registry$' --filter-application-name='^SYSTEM$' --filter-host-name='^THEHOST$' --warning-status='%{state} !~ /online/' --change-perfdata='.*#(.*),$1'
OK: Node 'Registry' with PID '173' from application 'SYSTEM' on host 'THEHOST' state is 'online', Log error: 0, Log warning: 0, Uptime: 4M 2w 2d 2h 40m, CPU usage: 0.00%, Heap usage: 46.10MB/254.00MB (18.15%) | 'node.log.error.count'=0;;;0; 'node.log.warning.count'=0;;;0; 'node.cpu.usage.percentage'=0.00%;;;0;100 'node.heap.usage.percentage'=18.15%;0:90;;0;100`

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-infor-ion-grid-restapi.md [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+</Tabs>
+
+### Règles de découverte
+
+#### Découverte de services
+
+| Nom de la règle                             | Description                                                                      |
+|:--------------------------------------------|:---------------------------------------------------------------------------------|
+| App-Infor-Ion-Grid-Restapi-Application-Name | Découvre les points d'application Infor ION Grid et supervise leur statut        |
+| App-Infor-Ion-Grid-Restapi-Node-Name        | Découvre les noeuds Infor ION Grid et supervise leur statut et leur statistiques |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-infor-ion-grid-restapi.md:41]:
```suggestion
| App-Infor-Ion-Grid-Restapi-Node-Name        | Découvre les noeuds Infor ION Grid et supervise leur statut et leurs statistiques |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-infor-ion-grid-restapi.md [hunk 3]
```diff
@@ -0,0 +1,393 @@
...
+| *nodes*#node.uptime.seconds        | s     |
+| *nodes*#node.cpu.usage.percentage  | %     |
+| *nodes*#node.heap.usage.percentage | %     |
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+*Specify prerequisites that are relevant. You may want to just provide a link\n\
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-infor-ion-grid-restapi.md:75]: No prerequisites!

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-infor-ion-grid-restapi.md (hunk 1 above)
Comment (cg-tw) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-infor-ion-grid-restapi.md:239]: you can use the example from the community PR if we are sure all the data is anonymized

File: pp/integrations/plugin-packs/procedures/applications-infor-ion-grid-restapi.md [hunk 4]
```diff
@@ -0,0 +1,392 @@
...
+</TabItem>
+</Tabs>
+
+### Discovery rules
+
+#### Service discovery
+
+| Rule name                                   | Description |
+|:--------------------------------------------|:------------|
+| App-Infor-Ion-Grid-Restapi-Application-Name | Discover anypoint of application Infor ION Grid and monitor their status |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-infor-ion-grid-restapi.md:39]:
```suggestion
| App-Infor-Ion-Grid-Restapi-Application-Name | Discover all application points for Infor ION Grid and monitor their status |
```

Comment (cg-tw) [reply pp/integrations/plugin-packs/procedures/applications-infor-ion-grid-restapi.md:39]: what is a "point d'application"?

File: pp/integrations/plugin-packs/procedures/applications-infor-ion-grid-restapi.md [hunk 5]
```diff
@@ -0,0 +1,392 @@
...
+| *nodes*#node.uptime.seconds        | s     |
+| *nodes*#node.cpu.usage.percentage  | %     |
+| *nodes*#node.heap.usage.percentage | %     |
+
+</TabItem>
+</Tabs>
+
+## Prerequisites
+
+*Specify prerequisites that are relevant. You may want to just provide a link\n\
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-infor-ion-grid-restapi.md:74]: Prerequisites

File: pp/integrations/plugin-packs/procedures/applications-infor-ion-grid-restapi.md [hunk 6]
```diff
@@ -0,0 +1,392 @@
...
+
+3. [Deploy the configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). The service appears in the list of services, and on the **Resources Status** page. The command that is sent by the connector is displayed in the details panel of the service: it shows the values of the macros.
+
+## How to check in the CLI that the configuration is OK and what are the main options for?
+
+Once the plugin is installed, log into your Centreon poller's CLI using the
+**centreon-engine** user account (`su - centreon-engine`). Test that the connector 
+is able to monitor a resource using a command like this one (replace the sample values by yours):
+
+```bash
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-infor-ion-grid-restapi.md:237]: FR and EN must be the same

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**